soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3956,6 +3959,14 @@ void DrawRandoEditor(bool& open) {
                                             "expected to be collected after getting Sun's Song.");
                         PaddedSeparator();
 
+                        // Shuffle Cows
+                        ImGui::Text(Settings::ShuffleCows.GetName().c_str());
+                        InsertHelpHoverText(
+                            "Enabling this will let cows give you items upon performing Epona's Song in "
+                            "front of them.");
```

Comment:
seems this is the same text used in 3drando's description *and* in 64rando's description, but i'm still not the biggest fan of it

mostly i don't like the `Enabling this will let` part. I think i'd prefer

```
Cows give a randomized item from the pool upon performing Epona's Song in front of them.
```
or 
```
Cows give a randomized item from the pool instead of milk the first time Epona's Song is played for them.
```
or something

doesn't need to change for this PR, just felt like getting some ideas down